core/src/main/resources/openapi/eth2signer.yaml

```diff
@@ -0,0 +1,92 @@
+openapi: 3.0.0
+info:
+  title: 'Eth2Signer'
+  description: 'Sign Eth2 Artifacts'
+  version: '1.0.0'
+  license:
+    name: 'Apache 2.0'
+    url: 'http://www.apache.org/licenses/LICENSE-2.0.html'
+
+servers:
+  - url: /
+  - url: http://localhost:9000/
+
+paths:
+  /signer/sign/{publicKey}:
+    post:
+      tags:
+      - 'Signing'
+      summary: 'Signs data for public key'
+      description: 'Signs data for the public key specified as part of the URL and returns the signature'
+      operationId: 'signForPublicKey'
+      parameters:
+      - name: 'publicKey'
+        in: 'path'
+        required: true
+        description: 'Public Key for which data to sign'
+        type: 'string'
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                signingRoot:
+                  type: string
+              example:
+                signingRoot: '0x48656c6c6f2c20776f726c6421'
+      responses:
+        '200':
+          description: 'hex encoded string of signature'
+          content:
+            application/json:
```

Comment:
response type is actually text/plain as the value is returned as is without any extra formatting